modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -55,3 +55,11 @@ def tearDown(self):
         self.logger.debug("All services killed.")
 
         super().tearDown()
+
+    def _global_param(self, param_name, default=None):
+        """Reads global parameter passed to the test suite."""
+        return self.test_context.globals.get(param_name, default)
+
+    def _global_int(self, param_name, default: int = None):
```

Comment:
1) any reason to start a name with "_"?
2) "default=None" or "int = None", please format the code.

Reply:
Hm. The autoformat says current typing is correct where as supposed fixes are not:
For 'default = None' - PEP 8: E251 unexpected staces...
For 'int=None' - PEP 8: E252: Missing whitespace aroung parameters.

Same from pylint.

The reason for _global_param is that it's like 'private' method. Not intended to call it outside